projects/packages/masterbar/src/profile-edit/bootstrap.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Bootstrap the WP.com User profile edit restriction.
+ *
+ * @package automattic\jetpack
```

Comment:
We want to change the package to `jetpack-masterbar`

Reply:
Fixed all package annotations with https://github.com/Automattic/jetpack/pull/37342/commits/d117431f364b2fb5b3df13cb65e859fce66b4981